airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -24,8 +24,12 @@
   dockerRepository: airbyte/source-amazon-seller-partner
   dockerImageTag: 0.2.5
   sourceType: api
-  documentationUrl: https://docs.airbyte.io/integrations/sources/amazon-seller-partner
```

Comment:
why did we remove this ?

Reply:
I think by mistake you have removed icon and documentation for amazon seller partner